docs/manual/default-ports.md

```diff
@@ -118,15 +118,16 @@ These defaults can ofcoarse be changed, but as we guarantee "sane, working defau
 | Hyperion-ng         |   json    |  36096   |                   |                                      |
 | Hyperion-ng         |  protobuf |  36097   |                   |                                      |
 | Hyperion-ng         |  boblight |  36098   |                   |                                      |
-| OpenLDAP            |  main     |  36099   |                   |                                      |
+| OpenLDAP            |  Main     |  36099   |                   |                                      |
 | OpenLDAP            |  https    |  36100   |                   |                                      |
-| pihole              |  main     |  36101   |                   |                                      |
-| Sogo                |  main     |  36102   |                   |                                      |
-| oscam               |  main     |  36103   |                   |                                      |
-| teamspeak3          |  main     |  36104   |                   |                                      |
+| pihole              |  Main     |  36101   |                   |                                      |
+| Sogo                |  Main     |  36102   |                   |                                      |
+| oscam               |  Main     |  36103   |                   |                                      |
+| teamspeak3          |  Main     |  36104   |                   |                                      |
```

Comment:
Services cannopt use caps, so also don't use caps here.

Reply:
While at it, I fixed the other also